docs/application/web/guides/applications/addon.md

```diff
@@ -0,0 +1,202 @@
+# Web Application Addon
+
+Web Application addons are software programs that run on the Web applications. They work as part of each Web application. You can use addons to add or customize the Web application behavior according to your needs or preferences in all commonly installed Web applications. Web technologies such as HTML, JavaScript, and CSS are used to build them.
+
+**Figure: Addon Concept**
+
+![Addon concept](./media/addon_concept.png)
+
+> [!NOTE]
+> This feature is supported from Tizen version 5.5.
+
+The main features of the service application include:
+
+- [Configuration](#configure)
+
+  You must have a manifest file and a main JavaScript file.
+
+- [Event](#event)
+
+  You can use several events from a Web application to get the timing to perform what you want using addon on the Web application.
+
+- [Mechanism](#mechanism)
+
+  You can understand how each addon works.
+
+- [Comparing to Chrome Extensions](#compare)
+
+  Addons look similar to Chrome Extensions. However, there are certain differences that you can understand in this section.
+
+- [API](#api)
+
+  To write addons, you can use the predefined set of APIs.  
+
+- [Packaging](#packaging)
+
+  To generate a package for an addon, you can use Tizen Studio. There is a separate tag for addon in config.xml.
+
+- [Management](#management)
+
+  To manage installed addons, you can use a separate Web application named Web App Addon Setting.
+
+<a name="configure"></a>
+## Configuration
+
+### Manifest file
+
+You must have a manifest file.
+
+The manifest file includes the metadata for an addon:
+
+- **name**: Name of the addon.
+- **version**: Version of the addon.
+-	**main**: Name of the main JavaScript.
+-	**settings**: Null (This is for future functionality)
+-	**description**: Description of the addon.
+
```

Comment:
Request you to please fix the spacing as follows:

- **name**: Name of the addon.
- **version**: Version of the addon.
- **main**: Name of the main JavaScript.
- **settings**: Null (This is for future functionality)
- **description**: Description of the addon.

Reply:
done. 
I'm using Visual Studio Code as an editor.
In it, I could have seen the issue you mentioned unfortunately.
However, using gedit, I found what you mentioned and fixed as you mentioned.

Reply:
@hyundukkim,

I understand. Thank you for making the change.